docker/docker-compose.yml

```diff
@@ -2,8 +2,8 @@ version: "3.8"
 services:
   webapp:
     build:
-      context: .
-      dockerfile: Dockerfile
+      context: ..
+      dockerfile: docker/Dockerfile
```

Comment:
Set the `context:` to `./docker` and leave the `dockerfile:` as it was you should be able to run `docker compose` from the project root and not have any issues.